tests/test_solvermode/test_models.py

```diff
@@ -5,12 +5,12 @@
 @pytest.mark.quick
 @pytest.mark.setup
 @pytest.mark.fluent_version("latest")
-def test_solver_models(load_mixing_elbow_mesh):
-    solver_session = load_mixing_elbow_mesh
+def test_solver_models(load_mixing_elbow_settings_only):
+    solver_session = load_mixing_elbow_settings_only
     models = solver_session.setup.models
-    assert not models.energy.enabled()
-    models.energy.enabled = True
     assert models.energy.enabled()
+    models.energy.enabled = False
```

Comment:
Any reason this changed, the logic was reversed before

Reply:
When loading a mesh file, energy is disabled by default, but the case file has energy transport turned on. To test changing the energy transport model I figured the way to solve this would be to test turning it off. We would still be checking the `True` and `False` states, just in the opposite order.